src/VisualStudio/Core/Def/Implementation/Snippets/AbstractSnippetInfoService.cs

```diff
@@ -113,35 +113,16 @@ private void PopulateSnippetCaches()
         {
             Debug.Assert(_expansionManager != null);
 
-            // If the expansion manager is an IExpansionManager, then we can use the asynchronous
-            // population mechanism it provides. Otherwise, getting snippet information from the
-            // IVsExpansionManager must be done synchronously on the UI thread.
-
-            // IExpansionManager was introduced in Visual Studio 2015 Update 1, but will be enabled
-            // by default for the first time in Visual Studio 2015 Update 2. However, the platform
-            // still supports returning the IVsExpansionManager if a major problem in the 
-            // IExpansionManager is discovered, so we must continue supporting the fallback.            
-
             var token = _waiter.BeginAsyncOperation(GetType().Name + ".Start");
 
-            var asyncExpansionManager = _expansionManager as IExpansionManager;
-            if (asyncExpansionManager != null)
-            {
-                // Call the asynchronous IExpansionManager API from a background thread
-                Task.Factory.StartNew(async () => await PopulateSnippetCacheAsync(asyncExpansionManager).ConfigureAwait(false),
-                                CancellationToken.None,
-                                TaskCreationOptions.None,
-                                TaskScheduler.Default).CompletesAsyncOperation(token);
+            // In Dev14 Update2+ the platform always provides an IExpansion Manager
+            var asyncExpansionManager = (IExpansionManager)_expansionManager;
+            // Call the asynchronous IExpansionManager API from a background thread
+            Task.Factory.StartNew(async () => await PopulateSnippetCacheAsync(asyncExpansionManager).ConfigureAwait(false),
```

Comment:
you can remove the "async, await, and ConfigureAwait" from this expression entirely.